conf/resinhup.conf

```diff
@@ -54,6 +55,7 @@ to_keep_files:
 ; persistent over updates.
 ; Only relative paths to boot partition.
 ignore_files:
```

Comment:
ignore_files section seems to be the same as the boot_whitelist. Why this duplication?